components/gallery/GalleryItemMoreActionBtn.vue

```diff
@@ -4,7 +4,7 @@
 
     <template #items>
       <NeoDropdownItem
-        v-if="currentGalleryItemImage.mimeType.includes('image')"
+        v-if="mimeType.includes('image') && ipfsImage && name"
```

Comment:
I think the `name` is not necessary because there is a default name for the image.